3_run_rsync/tasks/create-pod-service.yml

```diff
@@ -44,6 +44,31 @@
     src: "{{ mig_dest_ssh_public_key }}"
   register: ssh_public_key
 
+- name: "Get stunnel crt"
+  slurp:
+    src: "{{ stunnel_cert_filepath }}"
+  register: stunnel_crt
+
+- name: "Get stunnel key"
+  slurp:
+    src: "{{ stunnel_key_filepath }}"
+  register: stunnel_key
+
+- name: "Generate rsyncd password"
+  vars:
+    rand_password: "{{ lookup('password', '/dev/null length=15 chars=ascii_letters') }}"
+  set_fact:
+    rsync_password: "{{ rand_password }}"
+
+- set_fact:
```

Comment:
@alaypatel07 Instead of having to set a fixed password for debugging purposes, you can always pass `rsyncd_password` as an extra variable which will override the one you set through random password. This also removes the need of having an extra flag `debug_password`. Extra variables always take the highest precedence (even over set_fact).